middleware/fastcgi/fcgiclient.go

```diff
@@ -380,7 +380,10 @@ func (c *FCGIClient) Request(p map[string]string, req io.Reader) (resp *http.Res
 		if err != nil {
 			return
 		}
-		resp.Status = statusParts[1]
+		if len(statusParts) > 0 {
```

Comment:
no questions, sorry - was pulled out to other projects. change is already there - **if len(statusParts) > 1** it is